example/pubspec.lock

```diff
@@ -418,4 +418,4 @@ packages:
     source: hosted
     version: "2.2.1"
 sdks:
-  dart: ">=2.10.0-110 <2.11.0"
+  dart: ">=2.12.0-0 <3.0.0"
```

Comment:
Careful, I think 2.12 is beta.
Maybe better to stay in 2.10 (stable) for the moment.